### PR TITLE
test: export on change

### PR DESCRIPTION
**Pull request by nikku**
This way we are able to validate the resulting XML.

### CODE REVIEW

File: test/base/ModelerSpec.js [hunk 1]
```diff
@@ -84,6 +86,18 @@ describe('<BaseModeler>', function() {
       ...options
     });
 
+    modeler.on('commandStack.changed', function() {
+
+      setTimeout(function() {
+
+        modeler.saveXML({ format: true }).then(function(result) {
+          console.clear();
```
Comment (philippfromme) [inline test/base/ModelerSpec.js:94]: Clearing the console also swallows all potential errors. I'd say we're fine without it.

File: test/base/ModelerSpec.js [hunk 2]
```diff
@@ -84,6 +86,18 @@ describe('<BaseModeler>', function() {
       ...options
     });
 
+    modeler.on('commandStack.changed', function() {
+
+      setTimeout(function() {
```
Comment (philippfromme) [inline test/base/ModelerSpec.js:91]: Why not use `debounce`?

Comment (nikku) [reply test/base/ModelerSpec.js]: Using debounce right now.

File: test/base/ModelerSpec.js (hunk 1 above)
Comment (nikku) [reply test/base/ModelerSpec.js]: Removed the clear portion.

**Comment by nikku** [comment]
Please have another look.

I've also ensured we only add it during single start so that it does not spam our logs.

**Comment by nikku** [comment]
I think we can merge this given that I've simply incorporated your review feedback.

**Pull request merged**